osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/datetime/SetActivityCalendarCommandActivationExecutor.java

```diff
@@ -41,10 +48,21 @@ public MethodResultCode execute(
       throws ProtocolAdapterException {
 
     LOGGER.debug("ACTIVATING PASSIVE CALENDAR");
+    final CosemObject cosemObject;
+    try {
+      cosemObject =
+          this.objectConfigService.getCosemObject(
+              device.getProtocolName(),
+              device.getProtocolVersion(),
+              DlmsObjectType.ACTIVITY_CALENDAR);
+    } catch (final ObjectConfigException e) {
+      throw new ProtocolAdapterException(AbstractCommandExecutor.ERROR_IN_OBJECT_CONFIG, e);
+    }
+
     final MethodParameter method =
         new MethodParameter(
-            CLASS_ID,
-            OBIS_CODE,
+            cosemObject.getClassId(),
+            cosemObject.getObis(),
             METHOD_ID_ACTIVATE_PASSIVE_CALENDAR,
```

Comment:
This method id is also defined in ActivityCalendarMethod.ACTIVATE_PASSIVE_CALENDAR

Reply:
Will use this ActivityCalendarMethod.ACTIVATE_PASSIVE_CALENDAR instead